gulpfile.js

```diff
@@ -23,15 +23,15 @@ gulp.task('browser', function() {
     .pipe(gulp.dest('./Example/source'));
 });
 
-gulp.task('es6', function(){
+gulp.task('es6',_ => {
```

Comment:
@tarungarg546 Oh yes! That is because `./Example/test.js` is for me to test the code on browser. Apologies for it. Not sure if I should keep this in gulp file or upload `./Example/test.js` without API Key on Github.